authors/apps/articles/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class ArticlesConfig(AppConfig):
+    name = 'authors.apps.articles'
+
+    def ready(self):
+        from . import signals # noqa
```

Comment:
@cob04, why are you muting a flake8 issue using `# noqa`, also, why should you do an import in a class method?

Reply:
I'm activating the signals module so Django can pick it up.
This is the recommended way to register signals in Django but PEP8 will be a definitely be a blocker, so I muted that line.

Reply:
@cob04, can you kindly provide a resource backing this up.

Reply:
@abulojoshua1 let me search for one.

Reply:
[this book, **Chapter 3: Models > Structural Patterns > User Profiles**](https://www.packtpub.com/web-development/django-design-patterns-and-best-practices-second-edition) 

[https://simpleisbetterthancomplex.com/tutorial/2016/07/28/how-to-create-django-signals.html](https://simpleisbetterthancomplex.com/tutorial/2016/07/28/how-to-create-django-signals.html)